examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -240,6 +234,19 @@ void RegularGridDecomposition::exchangeMigratingParticles(AutoPasType &autoPasCo
       }
     }
   }
+  // sanity check: if the simulation is a closed system there should be no emigrants left at this point.
+  if (std::all_of(_boundaryType.begin(), _boundaryType.end(),
```

Comment:
Similar to what I wrote for Simulation.cpp, this should include reflecting boundaries. (Right?)